### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/create-pull-request/commit/a610f2d5f93019c22786d48d2ee9ca0667ada316#commitcomment-41543531

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/create-pull-request/commit/1f443bee9fdf59422d911225f7656890a4782ae9

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

**Comment by peter-evans** [comment]
Hi @jsoref 

Thanks for contributing!

Please exclude the `dist` directory. The contents of that are compiled from many dependencies that I have no control over.
I'm happy to accept the other changes once `dist/index.js` is excluded.

As for running the action in this repository, I think I'll pass for the moment, thank you. I've just recently started using the [Grammarly vscode plugin](https://marketplace.visualstudio.com/items?itemName=znck.grammarly) and it's helping me catch typos when I'm working on files pretty well!

File: dist/index.js [hunk 1]
```diff
@@ -645,7 +645,7 @@ const windowsRelease = release => {
 	// Server 2008, 2012, 2016, and 2019 versions are ambiguous with desktop versions and must be detected at runtime.
 	// If `release` is omitted or we're on a Windows system, and the version number is an ambiguous version
 	// then use `wmic` to get the OS caption: https://msdn.microsoft.com/en-us/library/aa394531(v=vs.85).aspx
-	// If `wmic` is obsoloete (later versions of Windows 10), use PowerShell instead.
+	// If `wmic` is obsolete (later versions of Windows 10), use PowerShell instead.
```
Comment (jsoref) [inline dist/index.js:648]: For my future reference:
https://github.com/sindresorhus/windows-release/blob/6f4a280599f8ac95725a35b471cfc0e2305aeb43/index.js#L32

File: dist/index.js [hunk 2]
```diff
@@ -1073,7 +1073,7 @@ function createOrUpdateBranch(git, commitMessage, base, branch, branchRemoteName
         // Save the working base changes to a temporary branch
         const tempBranch = uuid_1.v4();
         yield git.checkout(tempBranch, 'HEAD');
-        // Commit any uncomitted changes
+        // Commit any uncommitted changes
```
Comment (jsoref) [inline dist/index.js:1076]: I can't figure out where this code comes from.

File: dist/index.js [hunk 3]
```diff
@@ -2611,9 +2611,9 @@ function parse(options) {
 
   const omittedParameters = Object.keys(options).filter(option => urlVariableNames.includes(option)).concat("baseUrl");
   const remainingParameters = omit(parameters, omittedParameters);
-  const isBinaryRequset = /application\/octet-stream/i.test(headers.accept);
+  const isBinaryRequest = /application\/octet-stream/i.test(headers.accept);
```
Comment (jsoref) [inline dist/index.js:2614]: For my future reference:
https://github.com/octokit/endpoint.js/blob/dc03f51dbf580ba9f3a771346a3650ee8809e787/src/parse.ts#L43

File: dist/index.js [hunk 4]
```diff
@@ -2760,7 +2760,7 @@ module.exports = require("crypto");
 
 "use strict";
 
-// Older verions of Node.js might not have `util.getSystemErrorName()`.
```
Comment (jsoref) [inline dist/index.js:2763]: For my reference, this appears to originate here (this code is dead):
https://github.com/sindresorhus/execa/blob/575d464976c022cd127764e078066ba1e7d4dcdc/lib/errname.js#L2

I'm not sure how it arrived, although there is a stale copy here:
https://github.com/octokit/graphql-action/blob/5c1fea5240db8062f5a757c9c3d9a200ab82d970/dist/index.js#L3716

File: dist/index.js [hunk 5]
```diff
@@ -3006,7 +3006,7 @@ class Octokit {
       error: console.error.bind(console)
     }, options.log);
     this.hook = hook; // (1) If neither `options.authStrategy` nor `options.auth` are set, the `octokit` instance
-    //     is unauthenticated. The `this.auth()` method is a no-op and no request hook is registred.
```
Comment (jsoref) [inline dist/index.js:3009]: For my reference, this appears to come from here; https://github.com/octokit/graphql-action/blob/5c1fea5240db8062f5a757c9c3d9a200ab82d970/dist/index.js#L3023
originating here:
https://github.com/octokit/core.js/blob/4d96857fe87c57330fe024a2b0dbf7f1848cd769/src/index.ts#L123

File: dist/index.js [hunk 6]
```diff
@@ -6820,7 +6820,7 @@ function outside (version, range, hilo, options) {
       throw new TypeError('Must provide a hilo val of "<" or ">"')
   }
 
-  // If it satisifes the range it is not outside
```
Comment (jsoref) [inline dist/index.js:6823]: This appears to come from: https://github.com/npm/node-semver/blob/f56505b1c08856a7e6139f6ee5d4580f5f2feed8/ranges/outside.js#L35

File: dist/index.js [hunk 7]
```diff
@@ -7266,7 +7266,7 @@ function isDirectory(fsPath, useStat = false) {
 }
 exports.isDirectory = isDirectory;
 /**
- * On OSX/Linux, true if path starts with '/'. On Windows, true for paths like:
```
Comment (jsoref) [inline dist/index.js:7269]: This appears to be from: https://github.com/actions/toolkit/blob/1cc56db0ff126f4d65aeb83798852e02a2c180c3/packages/io/src/io-util.ts#L44

File: dist/index.js (hunk 2 above)
Comment (peter-evans) [reply dist/index.js]: This is my code. It's file `src/create-or-update-branch.ts`. This PR already corrects this occurrence.
The `dist/index.js` file is compiled from my code plus all of the dependencies it's using.

**Comment by jsoref** [comment]
@peter-evans: I dropped those commits/hunks.

**Comment by peter-evans** [comment]
@jsoref I still see `dist/index.js` in the diff.

**Comment by jsoref** [comment]
Oops. Sorry.

**Comment by peter-evans** [comment]
Thank you!

**Pull request merged**